drivers/spi/spi_ll_stm32.h

```diff
@@ -54,6 +57,21 @@ struct spi_stm32_data {
 };
 
 #ifdef CONFIG_SPI_STM32_DMA
+static inline uint32_t ll_func_get_reg_addr(SPI_TypeDef *spi, uint32_t location)
```

Comment:
Let's be a bit more specific: `ll_func_dma_get_reg_addr`

Reply:
renaming to "ll_func_dma_get_reg_addr"